go.mod

```diff
@@ -3,34 +3,37 @@ module github.com/grafana/xk6-websockets
 go 1.18
 
 require (
-	github.com/dop251/goja v0.0.0-20230919151941-fc55792775de
-	github.com/gorilla/websocket v1.5.0
+	github.com/dop251/goja v0.0.0-20240220182346-e401ed450204
+	github.com/gorilla/websocket v1.5.1
 	github.com/mstoykov/k6-taskqueue-lib v0.1.0
 	github.com/sirupsen/logrus v1.9.3
-	github.com/stretchr/testify v1.8.4
-	go.k6.io/k6 v0.47.1-0.20231012091148-de19a6a1bc53
-	go.uber.org/goleak v1.2.1
+	github.com/stretchr/testify v1.9.0
+	go.k6.io/k6 v0.50.0
```

Comment:
Out of the curiosity, why v50 and not v51? :thinking:

Reply:
aahh ... yes we have release v0.51.0 :sweat_smile:  I guess with me not being around for the release I didn't really update my mental note :facepalm:

Reply:
updated